2023/spring/Scheffler-MW@1230PM/Rose-Njuguna.md

```diff
@@ -0,0 +1,8 @@
+#Hello
```

Comment:
I think here you should have a space between the # and the text, making it something like:
`# Hello`
A cool thing to do before pushing markdowns is to run them through a online markdown editor like this one: https://dillinger.io/